examples/notebooks/responsibleaidashboard-diabetes-decision-making.ipynb

```diff
@@ -296,9 +296,12 @@
     "    print(\"X_train cols: {0}\".format(X_train.columns))\n",
     "\n",
     "    print(\"Training model\")\n",
+    "\n",
+    "    features_to_drop = [\"s3\", \"s5\"]\n",
```

Comment:
How were these chosen? I'm just asking since it may affect e2e tests. There are some that use specific features to create counterfactuals, for example. If the feature suddenly doesn't impact the model anymore due to it being dropped that could influence the outcome of the test case.

Reply:
Tested that these two features are not used for E2E testing. And got a green pass for E2E tests